engine/hunspell_table.py

```
@@ -8761,13 +8779,13 @@ def _change_line_direction(self, direction: str) -> None:
                              'the cursor does not seem to work with '
                              'gnome wayland input. ')
                 return
-            if not IMPORT_BIDI_ALGORITHM_SUCCESSFUL:
+            if bidi_algorithm is None:
                 LOGGER.debug(
                     '"import bidi.algorithm" didn’t work, '
```

Ruff (0.15.12)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)